docs/source/docker/getting_started/first_steps.rst

```diff
@@ -141,6 +146,10 @@ The command above does the following:
 - **stopping_condition.min_distance=0.2** would remove all samples which are
   closer to each other than 0.2. 
 
+By default the docker container does just create an output file with the selected
+filenames for you. You can also tell the solution to copy the selected files into
+the output folder by adding the parameter **dump_dataset=True** to the command.
+
```

Comment:
You could also describe it as follows:
**dump_dataset=default (False)** Only output the filenames of the images chosen by the sampler. Alternatively with `dump_dataset=True` the chosen images would be copied the output folder.